DesignerTest/ViewDesignTests.cs

```diff
@@ -74,5 +78,64 @@ public void ViewDesign_LoadProducts_Count()
             List<Product> products = ViewDesignViewModel.LoadProducts();
             Assert.AreEqual(3, products.Count);
         }
+
+        [Test]
+        [TestCase("0,0|500,0|500,500|0,500", ExpectedResult = 4)]
+        [TestCase("0,0|250,0|250,250|500,250|500,500|0,500", ExpectedResult = 6)]
+        public int ViewDesign_ConvertPosititionsToCoordinates_ShouldReturnList(string positions)
```

Comment:
Wellicht een beetje overbodige test? wordt momenteel al roomtests getest

Reply:
Ja zou 1 keer getest hoeven worden idd

Reply:
Gefixt in laatste commit